mlflow/pyfunc/__init__.py

```diff
@@ -946,12 +959,18 @@ def load_model(
 
     _add_code_from_conf_to_system_path(local_path, conf, code_key=CODE)
     data_path = os.path.join(local_path, conf[DATA]) if (DATA in conf) else local_path
+
+    if isinstance(model_config, str):
+        model_config = _validate_and_get_model_config_from_file(model_config)
+
+    # conf.get(MODEL_CONFIG) this should ALWAYS be a dict (no more saving as file!)
```

Comment:
@annzhang-db can we remove this inline comment?

Reply:
yes sorry 😅 this was also a comment for myself when I was working on it. forgot to delete 😓